.github/workflows/Publish.yml

```
@@ -1,342 +1,512 @@
+    shell: bash
 
 env:
-  DOTNET_VERSION: '8.x'
+  DOTNET_VERSION: "10.x"
   DOTNET_SKIP_FIRST_TIME_EXPERIENCE: true
   DOTNET_CLI_TELEMETRY_OPTOUT: true
+  DOTNET_NOLOGO: true
+
+  # IMPORTANT: analyze the same commit we report to Codacy
+  CHECKOUT_REF: ${{ github.event_name == 'pull_request' && github.event.pull_request.head.sha || github.sha }}
+  COMMIT_UUID:  ${{ github.event_name == 'pull_request' && github.event.pull_request.head.sha || github.sha }}
```

YAMLlint (1.38.0)

[error] too many spaces after colon

(colons)